arches/app/models/migrations/10999_add_principaluserid_to_resources.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.6 on 2019-12-13 11:56
+
+from django.conf import settings
+from django.db import migrations, models
+from django.db.models import deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "9979_update_stage_for_bulk_edit"),
```

Comment:
I have to admit, I have not had time to test this as an isolated change, so this seemed like the correct dependency, but would need confirmed.